src/chatdbg/util/config.py

```diff
@@ -17,8 +17,8 @@
 
 
 def _chatdbg_get_env(
-    option_name: str, default_value: Union[int, str, bool]
-) -> Union[int, str, bool]:
+    option_name: str, default_value: Union[int, bool, str]
+) -> Union[int, bool, str]:
```

Comment:
Funny one here: same types. Either order alphabetically or ignore?

Reply:
It's too bad it's so painful to specify that default_value will always have the same type as the return type.  I guess you could add @overrides to enumerate the cases in typerwriter when you see that sort of correlation.

Reply:
Sorting of types now implemented. Correlated `@overrides` would be a nice feature!